packages/react-data-grid/src/HeaderRow.tsx

```diff
@@ -50,11 +54,19 @@ export default class HeaderRow<R, K extends keyof R> extends React.Component<Hea
 
   getFilterableHeaderCell(column: CalculatedColumn<R>) {
     const FilterRenderer = column.filterRenderer || FilterableHeaderCell;
+    const { filters, onFiltersChange } = this.props;
+
+    const onChange = (value: unknown) => {
```

Comment:
Consider moving this outside and bind this so that the instance would be same in each render? I don't think we should convert this class component to functional component in this PR.

Reply:
I am going to cleanup header/filter rows in a future PR and convert them to functions